include/boost/geometry/strategies/cartesian/closest_points_projected_point.hpp

```diff
@@ -0,0 +1,290 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2019, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CARTESIAN_CLOSEST_POINTS_PROJECTED_POINT_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CARTESIAN_CLOSEST_POINTS_PROJECTED_POINT_HPP
+
+#include <boost/geometry/algorithms/detail/closest_points/result.hpp>
+
+#include <boost/geometry/formulas/cartesian.hpp>
+
+#include <boost/geometry/strategies/closest_points.hpp>
+#include <boost/geometry/strategies/cartesian/distance_pythagoras.hpp>
+#include <boost/geometry/strategies/cartesian/point_in_point.hpp>
+#include <boost/geometry/strategies/cartesian/intersection.hpp>
+
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace closest_points
+{
+
+template
+<
+    typename CalculationType = void,
+    typename Strategy = distance::pythagoras<CalculationType>
+>
+class projected_point
```

Comment:
This strategy is probably not needed here. I know you mimicked the `distance::projected_point` but I think the `Strategy` argument is not needed. Especially because other strategies are defined internally anyway. Plus, `projected_point` name could be `cartesian_point_segment` for consistency with other strategies you added in this PR.

Reply:
I changed the name but I am not sure I understand the issue with the Strategy, it is used internally to define some comparable strategies etc.